calculator.rb

```diff
@@ -0,0 +1,90 @@
+# prints a welcome message to the user
+print "Welcome to the calculating program! which operater you want to use \n \
+1. add(+)\n 2. substract(-)\n 3. multiply(*)\n 4. divide(/)\n"
+
+# prompt user to enter a math operater
+print "Please enter a math operater (name or a symbol): "
+math_operater = gets.chomp.downcase
+
+# creates an array with list of maths operator
+math_operators = ["+", "-", "/", "*", "add", "subtract", "multiply", "divide"]
+
+# ask user to input an maths operater until its valid
+until math_operators.include?(math_operater)
+    puts"Invalid math operator! Please try again."
```

Comment:
This is very minor, but I like having spaces inbetween my methods and strings, so I'd like a space between `puts` and the string, like `puts "Invalid math operator!"`